src/proto/Ethereum.proto

```diff
@@ -89,4 +89,7 @@ message SigningOutput {
     bytes v = 2;
     bytes r = 3;
     bytes s = 4;
+
+    // The payload part, returned separately
+    bytes payload = 5;
```

Comment:
This is transaction.data correct?

Reply:
Exactly.  For generic calls, data is supplied in input.  But for ERC20, 721, 1155 it is built from parameters supplied, so it is useful to return the built payload.

Reply:
Let's call it input(naming in Ethereum rpc, like r,s,v) or just data

Reply:
Renamed to data.